R/gh_response.R

```diff
@@ -0,0 +1,33 @@
+gh_process_response <- function(response) {
+  stopifnot(inherits(response, "response"))
+  heads <- headers(response)
+
+  content_type <- http_type(response)
+  if (is.null(content_type) || length(content_type) == 0) {
+    res <- ""
+  } else if (grepl("^application/json", content_type, ignore.case = TRUE)) {
+    res <- fromJSON(content(response, as = "text"), simplifyVector = FALSE)
+  } else {
+    if (grepl("^text/html", content_type, ignore.case = TRUE)) {
```

Comment:
This is the new scaled back handling of an html response (#37). I just throw a warning about html content and put it in the message, in case user wants to deal with it.